packages/terra-form/src/Field.scss

```diff
@@ -1,62 +1,80 @@
+@import '~terra-mixins';
+@import './cerner-mock-theme';
+
 :local {
-  .field,
-  .fieldset {
+  .field {
     border: 0;
-    margin-bottom: 1.428em;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
     padding: 0;
   }
 
+  // Remove this??
   @media screen and (min-width: 544px) {
-    .field-inline,
-    .fieldset-inline {
+    .field-inline {
       display: inline-block;
-      margin-right: 0.7143em;
+      margin-right: var(--terra-form-field-inline-margin-right, 0.7143em);
       vertical-align: top;
     }
   }
 
-  .field-required > .label:first-child,
-  .fieldset-required > .legend:first-child {
-    &::before {
-      color: #e50000;
-      content: '*';
-      display: inline-block;
-      padding-right: 0.357em;
-    }
+  .label-group {
+    font-family: var(--terra-form-field-label-font-family, 'Helvetica Neue');
+    font-size: var(--terra-form-field-label-font-size, 14px);
+    line-height: 1.29;
+    margin-bottom: var(--terra-form-field-label-margin-bottom, 0.357em);
+    text-align: left;
```

Comment:
Seems like we could have a themeable variable associated with `text-align` to account for label text placement. Thoughts?

Reply:
Base on the current mocks I've seen, text-alignment at this point will be left aligned. In the future there would be potential for right alignment, but I foreseeing any logic regarding that would need to be handled by the component itself.